examples/hpc/dla-driver/examples/mac_benchmark.rs

```diff
@@ -33,7 +36,7 @@ fn generate_random_array(buffer: &mut [i8], size: usize) {
 }
 
 fn generate_random_matrix(height: u32, width: u32, seed: u64) -> Vec<i8> {
```

Comment:
Consider using `usize` as the input parameter type, as that is what is required by this function. Otherwise you end up casting stuff back and forth without a good reason.

A good rule of thumb if you're considering whether something should be done by the function or its caller: a function should do as little work as possible. Applies to these types of casts as well.

Reply:
DLA changes were removed from the scope of this RP